docs/application/dotnet/guides/nui/resources.md

```diff
@@ -28,14 +28,14 @@ The two basic methods of handling image resources in NUI are as follows:
 
     imageVisual.SetTransformAndSize(imageVisualTransform, new Vector2(this.SizeWidth, this.SizeHeight));
 
-    //Create and register image visual.
+    //Create and register image visual
     VisualBase imageVisual = VisualFactory.Instance.CreateVisual(map);
     RegisterVisual(IMAGE_VISUAL_INDEX, imageVisual);
 
     /// Set the image visual depth index
     imageVisual.DepthIndex = IMAGE_VISUAL_INDEX;
     ```
-2. Load image resource in Image `Component`, for more information, see  [ImageView tutorial](imageview.md) page:
+2. Load image resource in image `Component`, for more information, see  [ImageView tutorial](imageview.md) page:
```

Comment:
@mihashco please let us know your opinion. 
Why component is in code style? 
If there is a specific reason to keep the component in code style, then ignore this comment else we can change it to:
```suggestion 
2. Load image resource in image component, for more information, see  [ImageView tutorial](imageview.md) page: